### PR TITLE
Features/update tests

### PR DESCRIPTION
**Pull request by 4en1x**
+some fixes

### CODE REVIEW

File: server/dao/impl/vacancies.dao.js [hunk 1]
```diff
@@ -48,12 +48,12 @@ class VacanciesDAO extends BasicDAO {
...
 
       await Promise.all(skills.map(async (skill) => {
-        const { id: skillId } = await SkillsDAO.instance.findByName(skill.name);
+        const { id: skillId } = await SkillsDAO.instance.findByName(skill);
 
         await this.connection.queryAsync({
           sql: `INSERT INTO vacancy_has_skills
                 (skill_id, vacancy_id, weight) VALUES (?, ?, ?)`,
-          values: [skillId, id, skill.weight],
+          values: [skillId, id, 5],
```
Comment (aaneitchik) [inline server/dao/impl/vacancies.dao.js]: Magic number, create a variable for it or add some comments

File: server/services/hirings.service.js [hunk 2]
```diff
@@ -27,8 +55,25 @@ function createHiringUpdateObject(reqBody) {
   };
 }
 
+function rebuildHiring(hiring) {
+  hiring.vacancyName = hiring.vacancyName
```
Comment (aaneitchik) [inline server/services/hirings.service.js]: hiring.vacancyName = hiring.vacancyName || 'Passed hiring process to the company'

File: server/services/hirings.service.js [hunk 3]
```diff
@@ -1,14 +1,42 @@
-const db = require('../dao/dao');
```
Comment (LeanidShutau) [inline server/services/hirings.service.js]: Why not to use the holder?

File: server/dao/impl/interviews.dao.js [hunk 4]
```diff
@@ -113,13 +113,14 @@ class InterviewsDAO extends BasicDAO {
 
     return super.find({
       fields: `i.${this.idField}, type, date, place, c.name, c.surname`,
-      basis: `${this.tableName} i
-              INNER JOIN ${feedbacksTableName} f
+      basis: `${feedbacksTableName} f
+              INNER JOIN ${this.tableName} i
```
Comment (LeanidShutau) [inline server/dao/impl/interviews.dao.js]: Wherefore?

**Pull request merged**